Zend/tests/type_declarations/intersection_types/implicit_nullable_intersection_type.phpt

```diff
@@ -1,10 +1,12 @@
 --TEST--
-Intersection types cannot be implicitly nullable
+Intersection types can be implicitly nullable as the others
 --FILE--
 <?php
 
-function foo(X&Y $foo = null) {}
+function foo(X&Y $foo = null): ?X&Y { return $foo; }
```

Comment:
I would revert the addition of `: ?X&Y`, as this test seems to be specifically intended for *implicit* nullability

Reply:
Agreed, removed, thanks for the review.

Reply:
leftover colon :s

Reply:
argh,fixed again